ipv-core/src/test/java/com/ibm/whc/deid/providers/masking/fhir/DateDependencyMaskingProviderTest.java

```diff
@@ -125,7 +125,7 @@ public void testMaskDayAfterRangeISO() throws Exception {
     maskingProvider.maskIdentifierBatch(maskingInputs);
     JsonNode maskedNode = getParentNodeFromMaskingInputs(maskingInputs);
 
-    assertEquals("2005-09-23T05:02:00Z", maskedNode.get("birthDate").asText());
+    assertEquals("2005-09-23T00:02:00-05:00", maskedNode.get("birthDate").asText());

```

Comment:
Is the ISO 8601 "Z" for Zulu/GMT support still around ? I forgot where it came up but the ISO8601 support came from a customer request way back.

Reply:
The ISO formatter accepts Z as an offset value.  I will check that we have some unit tests that are using it.